src/Komponents/ButtonCollection/Button.jsx

```diff
@@ -0,0 +1,15 @@
+import { NavLink } from "react-router-dom";
+
+import "./button.css";
+
+function Button({ color, value, to }) {
+  return (
+    <button className={color}>
```

Comment:
bad props naming for className
if it is className then it shouldn't be a color name